opencti-platform/opencti-graphql/src/database/engine.js

```diff
@@ -1474,11 +1476,13 @@ const computeQueryIndices = (indices, types) => {
           return READ_ENTITIES_INDICES;
         }
         // For relationships
```

Comment:
Question for l1471: why do we have READ_INDEX_INFERRED_ENTITIES for internal object types ? do we create internal objects with inferrence rules ? from what I see, the only inferred entities we create are incidents (ENTITY_TYPE_INCIDENT)

Reply:
Yes, the only type is ENTITY_TYPE_INCIDENT